lmdeploy/pytorch/engine/model_agent.py

```diff
@@ -161,6 +161,53 @@ def to_device(self, device: str):
         return AdapterInfo(**out_dict)
 
 
+@dataclass
+class VisionModelInputs:
+    """Vision model inputs."""
+    history_lengths: torch.LongTensor
```

Comment:
What's different between this history length and the one in ModelInputs

Reply:
same when there's no modelinputs.split, different on the other case.
this save the history_length fat modelinputs.split